core/src/main/java/com/sap/ai/sdk/core/common/ClientResponseHandler.java

```diff
@@ -87,7 +87,15 @@ private T parseSuccess(@Nonnull final ClassicHttpResponse response) throws E {
         tryGetContent(responseEntity)
             .getOrElseThrow(e -> exceptionFactory.build(message, e).setHttpResponse(response));
     try {
-      return objectMapper.readValue(content, successType);
+      final T value = objectMapper.readValue(content, successType);
+      val timeHeaders = response.getHeaders("x-upstream-service-time");
+      if (timeHeaders.length > 0) {
+        log.info("LLM request success with duration:{}", timeHeaders[0].getValue());
```

Comment:
(Question/Minor)

* Are all current **and future messages** to AI Core _LLM requests_? This is a generic/general response handling class 🤔 

* I wouldn't use "success", rather "completed". The user may have different understanding for "success".

* Can we add HTTP response size to the message? e.g. `13KB`
  `response.getEntity().getContentLength()`